app/views/users/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<% provide :title, t("titles.edit_title") %>
```

Comment:
dùng " thay cho ' trong toàn bộ file này

Reply:
Tại sao một mình trang edit.html.erb này phải đổi ạ? Các trang erb còn lại thì sao hả anh?